concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-101	101	cuatro	four	1500	FOUR
+Lionnet-1985-364-102	102	cuchillo	knife	1352	KNIFE
+Lionnet-1985-364-103	103	cuello	neck	1333	NECK
+Lionnet-1985-364-104	104	cuerno	horn	1393	HORN (ANATOMY)
+Lionnet-1985-364-105	105	cuerpo	body	1480	BODY
+Lionnet-1985-364-106	106	cuervo	crow	913	CROW
+Lionnet-1985-364-107	107	cueva	cave	1155	CAVE
+Lionnet-1985-364-108	108	culebra	snake	730	SNAKE
+Lionnet-1985-364-109	109	cuñada	sister in-law	2317	SISTER-IN-LAW
+Lionnet-1985-364-110	110	curar	cure	499	CURE
+Lionnet-1985-364-111	111	dar	give	1447	GIVE
+Lionnet-1985-364-112	112	darle vuelta	turn something	1445	TURN (SOMETHING)
+Lionnet-1985-364-113	113	decir	say	1458	SAY
+Lionnet-1985-364-114	114	defecar, estiercol	defecate, manure	676	SHIT (DEFECATE)
+Lionnet-1985-364-115	115	derramarse	spill	3130	SPILL (SOMETHING)
+Lionnet-1985-364-116	116	desear	desire	2117	DESIRE
+Lionnet-1985-364-117	117	desgastarse	become used, worn out		
+Lionnet-1985-364-118	118	desgranar	thresh	285	THRESH
+Lionnet-1985-364-119	119	diente	tooth	1380	TOOTH
+Lionnet-1985-364-120	120	diez	ten	1515	TEN
+Lionnet-1985-364-121	121	dormirse	sleep	1585	SLEEP
+Lionnet-1985-364-122	122	dos	two	1498	TWO
+Lionnet-1985-364-123	123	dulce	sweet	717	SWEET
+Lionnet-1985-364-124	124	elote	fresh corncob		
+Lionnet-1985-364-125	125	emborracharse	become drunk		
+Lionnet-1985-364-126	126	encino	oak	644	OAK
+Lionnet-1985-364-127	127	enemigo, bravo	enemy, brave	1358	ENEMY
+Lionnet-1985-364-128	128	enemigo, enfrentarse	enemy, confront		
+Lionnet-1985-364-129	129	enfriarse	cool down	3068	COOL DOWN
+Lionnet-1985-364-130	130	entrar	enter	749	ENTER
+Lionnet-1985-364-131	131	entumecerse	become numb	2216	BECOME NUMB
+Lionnet-1985-364-132	132	enviar	send	1127	SEND
+Lionnet-1985-364-133	133	escorpión	scorpion	1538	SCORPION
+Lionnet-1985-364-134	134	escribir	write	1672	WRITE
+Lionnet-1985-364-135	135	espalda	back	1291	BACK
+Lionnet-1985-364-136	136	espiga	spike	1249	SPIKE
+Lionnet-1985-364-137	137	espina, aguja	thorn, needle	124	THORN
+Lionnet-1985-364-138	138	espinarse	be stung		
+Lionnet-1985-364-139	139	esposa	wife	1199	WIFE
+Lionnet-1985-364-140	140	estar	be (at place)		
+Lionnet-1985-364-141	141	estar caliente / calentar	be warm	1341	HEAT UP
+Lionnet-1985-364-142	142	estar enfermo	be sick	1847	SICK
+Lionnet-1985-364-143	143	estar en recipiente	be (in container)		
+Lionnet-1985-364-144	144	estera	mat	195	MAT
+Lionnet-1985-364-145	145	estirar	stretch	214	STRETCH
+Lionnet-1985-364-146	146	estornudar	sneeze	1621	SNEEZE
+Lionnet-1985-364-147	147	estrella	star	1430	STAR
+Lionnet-1985-364-148	148	faja	belt	772	BELT
+Lionnet-1985-364-149	149	flechar	shoot arrow		
+Lionnet-1985-364-150	150	flor	flower	239	FLOWER
+Lionnet-1985-364-151	151	frente	forehead	123	FOREHEAD
+Lionnet-1985-364-152	152	frijol	bean	832	BEAN
+Lionnet-1985-364-153	153	fruta	fruit	1507	FRUIT
+Lionnet-1985-364-154	154	fumar	smoke	1689	SMOKE (INHALE)
+Lionnet-1985-364-155	155	ganar en el juego	win	866	WIN
```

Comment:
WIN is less specific, could also be a victory in a battle for example, etc. So this should be unmapped